cryptography/src/commonMain/kotlin/com/wire/kalium/cryptography/utils/ChaCha20Utils.kt

```diff
@@ -97,6 +95,8 @@ internal class ChaCha20Utils {
             val backupCoder = BackupCoder(userId, passphrase)
             val header = backupCoder.decodeHeader(encryptedDataSource)
 
+            // We need to read the ChaCha20 generated header prior to the encrypted backup file data to run some sanity checks
+
             val key = backupCoder.generateChaCha20Key(header).toUByteArray()
```

Comment:
maybe more meaningfull name would be good here and remove the need for a comment ? If not a better name would be nice anyway